pcsx2/GS/Renderers/HW/GSRendererHW.cpp

```diff
@@ -4370,17 +4384,20 @@ bool GSRendererHW::OI_BlitFMV(GSTextureCache::Target* _rt, GSTextureCache::Sourc
 	// Nothing to see keep going
 	return true;
 }
+bool GSRendererHW::IsBlendedOrOpaque()
+{
+	return (!PRIM->ABE || IsOpaque() || m_context->ALPHA.IsCdOutput());
+}
 
-bool GSRendererHW::IsConstantDirectWriteMemClear()
+bool GSRendererHW::IsConstantDirectWriteMemClear(bool include_zero)
 {
 	// Constant Direct Write without texture/test/blending (aka a GS mem clear)
 	if ((m_vt.m_primclass == GS_SPRITE_CLASS) && !PRIM->TME // Direct write
-		&& (!PRIM->ABE || IsOpaque() || m_context->ALPHA.IsCdOutput()) // No transparency
-		&& (m_context->FRAME.FBMSK == 0) // no color mask
+		&& (m_context->FRAME.FBMSK == 0 || (include_zero && m_vt.m_max.c.eq(GSVector4i(0)))) // no color mask
```

Comment:
`GSVector4i::zero()` (so it calls `_mm_setzero_epi32()` instead of `_mm_set1_epi32`).. it'll probably optimize to it anyway though